mk_mvc.mak

```diff
@@ -24,7 +26,7 @@ readtags.exe: readtags.c
 
 # Debug version
 dctags.exe: respmvc
-	cl /Zi -DDEBUG /Fe$@ @respmvc debug.c /link setargv.obj
+	cl /Zi -DDEBUG /Fe$@ @respmvc /link setargv.obj
 
```

Comment:
Ok, but I would like to focus on nib parser here.
Could you submit another pull request for debug.c releated things?
So you can make this commit about nim simpler.